students/smitco/lesson03/mailroom.py

```diff
@@ -44,25 +45,43 @@ def intro():
     else:
         return addressee    
 def donation(n):
-    dollars = input("What is the donation amount? ")
-    if int(dollars) < (10 ** 6):
-        position = donors.index(n)
-        a = amounts[position]
-        a = a + (int(dollars),)
-        amounts[position] = a
-        return dollars
+    if n == "quit":
+        print()
+        pass
```

Comment:
the pass statement here and line 56 have no effect; they should be removed.

Reply:
That makes sense looking back at it. I had pass in there originally, but I added the extra print line for aesthetics. I didn't think about not needing pass once I had the print statement in there. Thanks!